harvard-lts/CURIOSity
# Fix cookie

**Fix the stale cookie problem**
* * *

# What does this Pull Request do?
Alters the cookie config so it expires sooner so we don't hit a stale cookie issue.

# How should this be tested?

This can only be tested on dev as it is for cas cookies. It is running on dev. I like to test by doing a new private window , log in with harvard key, and then just leave that window open in the background. IN about 1-1.5 hours come back and try to do a staff function (like go to manage users) and you should get a flash that your login expired and you need to log in again

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? n
- integration tests? n

# Interested parties
Tag (@ mention) interested parties
